src/figuresClasses.ts

```diff
@@ -1,19 +1,105 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea: () => number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string;
 
-}
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
 
-export class Circle {
+  constructor(
```

Comment:
you can use public modificator in this and nex cases, it's make you'r code shorter